src/Api/source.ts

```diff
@@ -0,0 +1,38 @@
+import { Client } from '../client';
+import { CreateSource } from '../interfaces/source';
+import { ResponseHelper } from '../utils/helpers/response-helper';
+
+export class Source {
+    private client: Client;
+
+    constructor(client: Client) {
+        this.client = client;
+    }
+
+    async all(query?: any) {
+        try {
+            const { data } = await this.client.httpGet(`/sources`, query);
+            return data;
+        } catch (error) {
+            ResponseHelper.handleErrors(error);
+        }
+    }
+
+    async create(attributes: CreateSource, query?: any) {
+        try {
+            const { data } = await this.client.httpPost(`/sources`, attributes, query);
+            return data;
+        } catch (error) {
+            ResponseHelper.handleErrors(error);
+        }
+    }
+
+    async find(id: string, query?: any) {
+        try {
+            const { data } = await this.client.httpGet(`/sources/${id}`, query);
+            return data;
+        } catch (error) {
+            ResponseHelper.handleErrors(error);
+        }
+    }
+}
```

Comment:
The delete method is missing here. There's an endpoint for deleting sources @horlah